pytest.ini

```diff
@@ -1,2 +1,3 @@
 [pytest]
+testpaths = tests
```

Comment:
Can we address this in the separate PR please?

Reply:
Done in https://github.com/django-oscar/django-oscar-paypal/pull/251